common/src/main/java/com/microsoft/identity/common/internal/ui/webview/challengehandlers/SwitchBrowserHandler.kt

```diff
@@ -97,9 +99,22 @@ class SwitchBrowserHandler(
         Logger.info(methodTag, "Launching switch browser request on browser: ${browser.packageName}")
         browserIntent.setPackage(browser.packageName)
         browserIntent.setData(switchBrowserChallenge.uri)
+        enableSwitchBrowserFlagOnWebViewFragment()
         activity.startActivity(browserIntent)
     }
 
+    /**
+     * Enable the flag to indicate the WebViewAuthorizationFragment is waiting for the switch browser resume endpoint.
+     */
+    private fun enableSwitchBrowserFlagOnWebViewFragment() {
```

Comment:
The name of this method sound like we are enabling a feature - which isn't true because at this point in code switch browser is already enabled. 

Perhaps we can name it `declareSwitchBrowserFlowOnFragment` or `setIsSwitchBrowserFlowOnFragment`

Reply:
rename and reworked, see SwitchBrowserRequestHandler.isChallengeHandled